project1/server/agent.cpp

```diff
@@ -182,15 +182,15 @@ namespace server {
         }
 
         // monitor for termination request for get and put commands
-        uint16_t total_bytes_sent = 0;
+        uint32_t total_bytes_sent = 0;
         bool terminated = false;
 
         // continue until we've sent the entire message
         while (total_bytes_sent < outgoing_message_buffer_.size()) {
             if (!terminated) {
-                const uint32_t kBytesToSend = std::min( (int) outgoing_message_buffer_.size() - total_bytes_sent, 1000);
+                const uint32_t kSendChunkSize = std::min((int) outgoing_message_buffer_.size() - total_bytes_sent, kSendChunkSize);
```

Comment:
Ah, whoops, sorry. I got confused by the variable naming. `kBytesToSend` is a fine name for the result of the `std::min` call. I guess what I meant to suggest is that you add a new `constexpr` variable called `kChunkSize` which is set to 1000, kind of like what you did on the receiving side.